README.md

```diff
@@ -177,6 +177,8 @@ dependencies {
 Don't know how to do that?? Take a look at
 the [application class example](./app/src/main/java/io/wax911/emojifysample/App.kt)
 
+**In order for EmojiInitilizer to work you need to add in your build.gradle `implementation "org.jetbrains.kotlinx:kotlinx-serialization-json:1.6.2"`
+
```

Comment:
I liked the initial idea of having `api()` for gradle, any reason why you chose to advice consumers to add `implementation`?

Reply:
Yes, the main difference between `api` and `implementation` is that when you use `api` it'll bring this library to the user project as well, but `implementation` is used the library will be only available in emojify.
There could be 2 kinds of "issue":
1. If user is using a different parsing library (like in my case) I'll have both Moshi and Kotlinx library in my project.
2. Users can have unexpected error if he uses a different version of kotlinx-serialization of the one used in emojify.

I have made a default KotlinxSerialization in order to not create breaking change, but even that is a bit of an issue as explained the in 2.